.travis.yml

```diff
@@ -1,2 +1,6 @@
 language: scala
-script: sbt "^ publish-local" "^ scripted"
+script: sbt "publish-local"
```

Comment:
This no longer runs the scripted tests in the travis build. We still need the scripted tests.